community/kernel/src/main/java/org/neo4j/kernel/configuration/ssl/LegacySslPolicyConfig.java

```diff
@@ -54,4 +59,15 @@
     public static final Setting<File> tls_key_file =
             derivedSetting( "unsupported.dbms.security.tls_key_file", certificates_directory,
                     certificates -> new File( certificates, "neo4j.key" ), PATH );
+
+    @Internal
+    @Description( "Default encryption protocol used for legacy SSl policy." )
```

Comment:
It is a bit wrong to mention default in the descriptions and names here, since you are overriding a default by using the setting. Applies to all settings and descriptions.

It is also not encryption protocol, but SSL/TLS protocol versions. Since most of the things here say TLS I think it is better perhaps to stick to that only.